dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,33 +1,45 @@
 -# English only for now
 - if language == "en"
 
-  -# Never show the survey.
-  - if false
+  -# US only
+  - if request.location.try(:country_code) == 'US' || (!Rails.env.production? && request.location.try(:country_code) == 'RD')
 
     -# If signed in, and user hasn't already filled out this survey
-    - if current_user && !current_user.survey2015_value
+    - if current_user && true # !current_user.survey2016_value
 
       -# If at least 13, and account is at least 14 days old
-      - if !current_user.under_13? && (DateTime.now - current_user.created_at.to_datetime) >= 14
+      - if !current_user.under_13? && true # (DateTime.now - current_user.created_at.to_datetime) >= 14
```

Comment:
Again, I'm not a fan of the && true. Also, update the comment to reflect that we aren't demanding the account be fourteen days old.